docs/CeedlingPacket.md

```diff
@@ -985,6 +985,40 @@ that you execute on target hardware).
 
   **Default**: FALSE
 
+  ---
+
+  **Note:**
+
+    The configuration can be combined together with
+    
+    ```
+    :test_runner:
+        :cmdline_args: true
+    ```
+
+    After setting **cmdline_args** to **true**, the debuger will execute each of test
+    case from crashing test file separately. The exclude and include test_case patterns will
+    be applied, to filter execution of test cases.
+
+    The .gcno and .gcda files will be generated and section of the code under test case
+    causing segmetation fault will be omitted from Coverage Report.
+
+    The default debugger (**gdb**)[https://www.sourceware.org/gdb/] can be switch by any other
+    via setting new configuration under tool node in project.yml. At default set to:
+
+    ```
```

Comment:
Not sure about rest of the file, but we might enable highlighting

```suggestion
    ```yaml
```

Reply:
Hi @Letme ,

Thank you for pointing to this. The language code mark was added as you suggested.